redis/src/main/scala/zio/redis/RedisExecutor.scala

```diff
@@ -2,17 +2,21 @@ package zio.redis
 
 import zio._
 import zio.logging._
+import zio.schema.codec.Codec
 
 object RedisExecutor {
   trait Service {
+
     def execute(command: Chunk[RespValue.BulkString]): IO[RedisError, RespValue]
+
+    def execute[In: Input, Out: Output](name: String, in: In): IO[RedisError, Out]
```

Comment:
As suggested in the previous comment, let's drop this method and replace it with:

```scala
def codec: Codec
```